### PR TITLE
feat(components): Add a validation hint to the Select

### PR DESCRIPTION
**Pull request by connor-baer**
Addresses #450.

## Purpose

Display a validation hint on a Select component, e.g. when the field is required.

## Approach and changes

The Select component is similar in shape to the Input component, thus we decided to adopt the same icon and placement of the validation hint there.

I've started to move (shared) SVG icons into `src/icons/` so that they can be easily replaced by our upcoming icon library down the line.

_Invalid:_

_With validation hint:_

_Invalid with validation hint:_

## Definition of done

* [x] Development completed
* [x] Reviewers assigned
* [x] Unit and integration tests
* [x] Meets minimum browser support
* [x] Meets accessibility requirements

### CODE REVIEW

File: src/components/Select/Select.js [hunk 1]
```diff
@@ -26,7 +26,9 @@ import {
 } from '../../util/shared-prop-types';
 import { textMega, disableVisually } from '../../styles/style-helpers';
 
-import { ReactComponent as ArrowsIcon } from './arrows.svg';
+import { ReactComponent as ArrowsIcon } from '../../icons/arrows.svg';
```
Comment (voronianski) [inline src/components/Select/Select.js]: I think we need to consider adding support for some shorthand aliases like `~/` with https://github.com/tleunen/babel-plugin-module-resolver as we did on the website.

Comment (connor-baer) [reply src/components/Select/Select.js]: I agree in principle, but I'd create a proposal issue first to get consensus from the rest of the web chapter.

Comment (voronianski) [reply src/components/Select/Select.js]: absolutely!

**Pull request merged**